app/Jobs/ImportItemsJob.php

```diff
@@ -69,8 +69,9 @@ public function handle()
 
             $item->save();
 
-            if ($item->code && $exhibition_ids->contains(Arr::get($record, 'fields.Výstava.0'))) {
-                $item->code->exhibition_id = Arr::get($record, 'fields.Výstava.0');
+            $exhibition_id = Arr::get($record, 'fields.Výstava.0');
+            if ($item->code && ($all_exhibition_ids->contains($exhibition_id) || $exhibition_id == null)) {
+                $item->code->exhibition_id = $exhibition_id;
                 $item->code->save();
             }
```

Comment:
This check feels a little surprising to me, but I think it's fine either way.

```suggestion
            if (!$item->code) return;
            $item->code->exhibition_id = $exhibition_id;
            $item->code->save();
```